### DISCUSSION TITLE
Update the look and feel of the page

### DISCUSSION THREAD

**Discussion by alvarolorentedev** [Minimum-CD/cd-manifesto#323] [Ideas]
Hello @Minimum-CD team, awesome work with the manifesto, I really love the work done.

I am curious if on your thoughts of a revamp of the UX with a framework like [docusaurus](https://docusaurus.io/). 
This allows the page to be a bit:
 - more malleable, as more people on the field know react than the templating system of hugo.
 - Have an up-to-date look & feel.

I would be happy to do a PR if this makes sense.

**Comment by bdfinst** [comment]
Strange. You'd think it would indicate a new discussion. None of us were alerted.
Conversation moved to PR: [324](https://github.com/Minimum-CD/cd-manifesto/pull/324)
